pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "pygui"
+version = "0.1.0"
+description = "Reactive state management for Python"
```

Comment:
I guess we need to think of something for this :)

Reply:
I've changed it to be "Reactive rendering for Python" until we can think of something better.